pkg/hhfab/testing.go

```
@@ -2181,19 +2212,21 @@ func checkIPerf(ctx context.Context, opts TestConnectivityOpts, iperfs *semaphor
 
 		if iPerfsMinSpeed > 0 {
 			if report.End.SumSent.BitsPerSecond < iPerfsMinSpeed*1_000_000 {
-				return fmt.Errorf("iperf3 send speed too low: %s < %s", asMbps(report.End.SumSent.BitsPerSecond), asMbps(iPerfsMinSpeed*1_000_000))
+				ie.ClientMsg = "iperf3 send speed too low"
+
+				return errors.New(ie.ClientMsg)
 			}
 			if report.End.SumReceived.BitsPerSecond < iPerfsMinSpeed*1_000_000 {
-				return fmt.Errorf("iperf3 receive speed too low: %s < %s", asMbps(report.End.SumReceived.BitsPerSecond), asMbps(iPerfsMinSpeed*1_000_000))
+				ie.ClientMsg = "iperf3 receive speed too low"
+
+				return errors.New(ie.ClientMsg)
 			}
 		}
 
 		return nil
 	})
 
 	if err := g.Wait(); err != nil {
```

As mentioned before, completely ignoring err from g.Wait() means some cleanup needed